site/content/docs/main/api-types/backup.md

```diff
@@ -93,7 +93,9 @@ spec:
   # all namespace-scoped resources are included. Optional.
   # Cannot work with include-resources, exclude-resources and include-cluster-resources.
   includedNamespaceScopedResources: {}
-  # Individual objects must match this label selector to be included in the backup. Optional.
+  # Individual objects must match this label selector to be included in the backup.
+  # If matched object is Namespace, the namespace will act as IncludedNamespaces, and all resources in the namespace are included in the backup.
+  # Optional.
   labelSelector:
```

Comment:
should the CRD description be updated? 
https://github.com/vmware-tanzu/velero/blob/dce97770cd28a716de95716cfe3816019c142f02/pkg/apis/velero/v1/backup_types.go#L88-L90

Reply:
done.